USAGOV-800: Add custom twig files for correct state directory URLs

akf opened the pull request:
<!--- Provide a general summary of your changes in the title above -->
## Jira Task
<!--- Provide a link to the Jira ticket -->
https://cm-jira.usa.gov/browse/USAGOV-800

## Description
<!--- Describe your changes in detail -->
Renamed the custom twig file for /states so that it will work on /state-governments, and added a custom twig file for the /gobiernos-estatales path. 

It's also necessary to make a change in content: Change the path alias of your /states page to /state-governments, and add a page for the Spanish directory: 

* path alias: /gobiernos-estatales (drupal will prepend /es to this)
* Check the "Custom twig content" box
* Language: Espanol
* Page type: Standard Page
* Copy over the relevant text from mothership
* Menu: Include in menu, select "Acerca de EE. UU. y directorios del Gobierno" as parent.

## Checklist for the Developer
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask for help! -->
- [ ] A link to the JIRA ticket has been added above.
- [ ] The JIRA ticket identifies the desired result of this change.
- [ ] The JIRA ticket contains clear acceptance criteria.
- [ ] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [ ] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

akf committed:
USAGOV-800-change-states-url: Add custom twig files for correct state directory URLs

IsabelLaurenceau committed:
Merge branch 'dev' into USAGOV-800-change-states-url

IsabelLaurenceau commented:
This works however I'm not approving it because the breadcrumbs are now broken for the states. I don't think that should be a separate ticket because the breadcrumbs depend on the path to display but if you feel it should be I can approve this. 
See breadcrumb.html.twig & breadcrumbs_for_states.html.twig

akf committed:
USAGOV-800-change-states-url: Update breadcrumbs for State directory pages

akf commented:
I've updated the breadcrumbs! Since the "state selector" pages should be in the menus, I've removed the clauses for them from the breadcrumb twig files as well. The standard breadcrumb code is working fine for those now.

IsabelLaurenceau reviewed (approved):
Works as expected now
